fv3_cap.F90

```diff
@@ -1041,47 +1095,19 @@ subroutine ModelAdvance_phase2(gcomp, rc)
 
     time_elapsed  = currtime - starttime
     na = nint(time_elapsed/timeStep)
-
-    ! if(mype==0) print *,'in fv3_cap,in model run, advance2,na=',na
+    call ESMF_TimeIntervalGet(time_elapsed, h=nfh, m=nfm, s=nfs, rc=rc)
+    nfseconds = nfh*3600.+ nfm*60. + nfs
```

Comment:
`3600.` should be just `3600`, `60.` should be `60`, since `nfseconds`, `nfh`, `nfm` and `nfs` are all integers. No need to have the RHS expression implicitly converted to real, and then converted back to integer.

Reply:
I think ESMF_TimeIntervalGet can be called like:

`    call ESMF_TimeIntervalGet(time_elapsed, s=nfseconds, rc=rc)
`

to return the seconds directly.

Reply:
Yes. it's good to use s=nfseconds, Done.